src/main.js

```diff
@@ -6,7 +6,7 @@ const logToFile = logger.logToFile.bind(logger);
 const seed = seeder.seed.bind(seeder);
 const setDefinitions = seeder.setDefinitions.bind(seeder);
 
-export {
+export default {
   setLoggingLevel,
   logToFile,
   seed,
```

Comment:
but now we cant import {seed } from saloon

Reply:
Updated `main.js` exports